### PR TITLE
validated syntax, added additional SFIA schema properties

### PR DESCRIPTION
**Pull request by GodsonLeigh**
Added SFIA scheme codes, scheme ID. 
Validated syntax

### CODE REVIEW

File: docs/asyncapi.yaml [hunk 1]
```diff
@@ -620,6 +654,10 @@ components:
       format: uuid
       description: GUID for a specific event reference
       example: 931e1a76-cff2-43ef-bfe5-82742eeeae68
+    RemainingAmount:
+      type: number
+      description: Amount left to be paid
+      example: 750
     SBI:
```
Comment (AbidemiAdio) [inline docs/asyncapi.yaml:661]: SBI is not Enum

File: docs/asyncapi.yaml [hunk 2]
```diff
@@ -641,6 +679,7 @@ components:
       example: 80001
       enum:
         - 80001 (Arable and horticultural soils Introductory)
+        - 80299 (SFI Accelerated Payment)
     SchemeId:
```
Comment (AbidemiAdio) [inline docs/asyncapi.yaml:683]: Should be enum

File: docs/asyncapi.yaml (hunk 1 above)
Comment (GodsonLeigh) [reply docs/asyncapi.yaml:661]: SBI:
      type: number
      description: Single Business Identifier
      minimum: 105000000
      maximum: 999999999
      example: 123456789
      
      SBI is given type: number

File: docs/asyncapi.yaml (hunk 2 above)
Comment (GodsonLeigh) [reply docs/asyncapi.yaml:683]: SchemeCode:
  type: string
  description: Unique funding option code
  example: 80001
  enum:
    - 80001 (Arable and horticultural soils Introductory)
    - 80299 (SFI Accelerated Payment)
    
    SchemeCode is enum using the code and descriptions provided

**Review by AbidemiAdio — APPROVED** [review]
QED

**Pull request merged**